tests/integration/spark/test_reducer_merge.py

```diff
@@ -218,6 +218,8 @@ def test_distributed_snapshot(self):
 
         # Retrieve list of file from the snapshotted PyRDF.RDataFrame
         input_files = snapdf.proxied_node.get_inputfiles()
-        # Remove unnecessary .root files
+        # Check that the intermediary .root files were created
+        # Then remove them because they are not necessary
         for filename in input_files:
+            self.assertTrue(os.path.exists(filename))
```

Comment:
Sorry, I think I did not explain my point correctly, I meant that would be nice to check that files are indeed created and they have the name we are setting for each range, so if in the test you know there will be two files with the following names:

- snapFile_0_4.root
- snapFile_5_9.root

then check that exact name. Therefore you may use a manually created list of files rather than the list of inputfiles extracted from the proxied node.

Reply:
Well better that I didn't get what you meant before, because now I have also noticed that the filename in `CallableGenerator` is retrieved only if in the form "filename.root"